apps/src/lib/kits/maker/boards/microBit/MicroBitConstants.js

```diff
@@ -92,3 +94,10 @@ for (let i = 0; i < 5; i++) {
   }
 }
 export const ALL_LEDS = allLeds;
+
+export const MICROBIT_FIRMATA_V1_URL = `${DOWNLOAD_PREFIX}microbit-firmata-v1-ver1.2.hex`;
+export const MICROBIT_FIRMATA_V2_URL = `${DOWNLOAD_PREFIX}microbit-firmata-v2-ver1.2.hex`;
+export const MICROBIT_IDS_V1 = ['9900', '9901'];
+export const MICROBIT_IDS_V2 = ['9903', '9904', '9905', '9906'];
+export const MICROBIT_V1 = 'v1';
```

Comment:
If you return the `firmataUrl` directly, it looks like you won't need these constants at all.

Reply:
I think I'll go ahead and keep the constants - thanks!